examples/classification/plot_classification_probability.py

```diff
@@ -56,40 +58,37 @@
 
 n_classifiers = len(classifiers)
 
-plt.figure(figsize=(3 * 2, n_classifiers * 2))
-plt.subplots_adjust(bottom=0.2, top=0.95)
-
-xx = np.linspace(3, 9, 100)
-yy = np.linspace(1, 5, 100).T
-xx, yy = np.meshgrid(xx, yy)
-Xfull = np.c_[xx.ravel(), yy.ravel()]
-
-for index, (name, classifier) in enumerate(classifiers.items()):
-    classifier.fit(X, y)
-
-    y_pred = classifier.predict(X)
+fig, axes = plt.subplots(
+    nrows=n_classifiers, ncols=len(iris.target_names), figsize=(10, 16)
```

Comment:
I think you should increase the width or the height of this plot: the rendering of this figure makes the subplots too small compared to the version on the `main` branch.